frog/imports/api/activities.js

```diff
@@ -39,50 +50,50 @@ export const addActivity = (activityType, data = {}, id, groupingKey) => {
   }
 };
 
-export const duplicateActivity = activity =>
+export const duplicateActivity = (activity: ActivityDbT) =>
   Activities.insert({
     ...activity,
     _id: uuid(),
     data: { ...activity.data, name: activity.data.name + ' (copy)' }
   });
 
-export const addGraphActivity = params =>
+export const addGraphActivity = (params: Object) =>
```

Comment:
What should params be here? Is it the data: {} object, or other things? If we know what it contains, we could be more specific than Object.